src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -452,6 +471,76 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
     });
   }
 
+  handlePromptDeletionOfDeadlines(submissionEndTimestamp: number): Observable<boolean> {
```

Comment:
To echo another previous point, yep am a bit confused by this naming - also it seems somewhat unclear just from reading this what the boolean return value is about (which seems to be whether to proceed with updating the feedback session)